pkg/namesgenerator/names-generator.go

```diff
@@ -281,7 +281,7 @@ var (
 		// Seymour Roger Cray was an American electrical engineer and supercomputer architect who designed a series of computers that were the fastest in the world for decades. https://en.wikipedia.org/wiki/Seymour_Cray
 		"cray",
 
-		// This entry reflects a husband and wife team who worked together:
+		// This entry reflects a spouse and spouse team who worked together:
```

Comment:
A "spouse and spouse" feels a bit awkward in this sentence, and given that this is about specific people, should probably be fine as-is.